tests/tr1/tests/type_traits5/test.cpp

```diff
@@ -19,6 +19,8 @@
 #define DOUBLE_ALIGN  4
 #define LDOUBLE_ALIGN 4
 
+#define CHECK_TYPEX(...) static_assert(STD is_same_v<__VA_ARGS__>, "BOOM")

```

Comment:
can we say something more informative than boom?

Reply:
Isn't the message kind of redundant as instantiation error already has that informative information?

Reply:
I'll change this message to "Something is wrong" :trollface:.

Reply:
> Isn't the message kind of redundant as instantiation error already has that informative information?

It's annoying when you're browsing through a log of test failures and you need to dig out the sources of some obscure test to figure out why the `static_assert` on line 467 is failing.

Reply:
Ah, did not know the expression isn't included in the message.

I'm having deja vu, I think I've said that before...